DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: janitor
 Title: Simple Tools for Examining and Cleaning Dirty Data
-Version: 0.2.1.9000
+Version: 0.3
```

Comment:
Never mind! Ignore my previous comment. I now think it's fine to keep the dev version at 0.2.1.9. Sorry for confusion.